### PR TITLE
[Issue #17][UI] messenger

### LINKED ISSUE #17: issue 9
image

- width 768px
image

- width 414px
image

### PR DESCRIPTION
**Pull request by hapo-lamnq**
close #17

### CODE REVIEW

**Comment by hapo-sonnx** [comment]
LGTM

File: javascript/messenger.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+$(document).ready(function () {
+    $("#messenger").click(function () {
```
Comment (hapo-nghialuu) [inline javascript/messenger.js]: chỉnh js indent là 2 spaces nhé

**Comment by hapo-sonnx** [comment]
LGTM

**Review by hapo-nghialuu — APPROVED** [review]

**Comment by hapo-nghialuu** [comment]
LGTM

**Pull request merged**